README.md

```diff
@@ -2,11 +2,876 @@
+
+# Manual Instrumentation Guide
+
+## Create two Spring Projects
+
+Using the [spring project initializer](https://start.spring.io/), we will create two spring projects.  Name one project `FirstService` and the other `SecondService`. Make sure to select maven, Spring Boot 2.3, Java, and add the spring-web dependency. After downloading the two projects include the OpenTelemetry dependencies and configuration listed below. 
+
+## Setup for Manual Instrumentation
+
+Add the dependencies below to enable OpenTelemetry in `FirstService` and `SecondService`. The Jaeger and LoggerExporter packages are recommended for exporting traces but are not required for this section. As of May 2020, Jaeger, Zipkin, OTLP, and Logging exporters are supported by opentelemetry-java. Feel free to use whatever exporter you are most comfortable with. 
+
+### Maven
+ 
+#### OpenTelemetry
+```xml
+<dependency>
+	<groupId>io.opentelemetry</groupId>
+	<artifactId>opentelemetry-api</artifactId>
+	<version>0.5.0</version>
+</dependency>
+<dependency>
+	<groupId>io.opentelemetry</groupId>
+	<artifactId>opentelemetry-sdk</artifactId>
+	<version>0.5.0</version>
+</dependency>	
+<dependency>
+    <groupId>io.grpc</groupId>
+    <artifactId>grpc-context</artifactId>
+    <version>1.24.0</version>
+</dependency>
+
+```
+
+#### LoggerExporter
+```xml
+<dependency>
+	<groupId>io.opentelemetry</groupId>
+	<artifactId>opentelemetry-exporters-logging</artifactId>
+	<version>0.5.0</version>
+</dependency>
+```
+
+#### JaegerExporter
+```xml
+<dependency>
+	<groupId>io.opentelemetry</groupId>
+	<artifactId>opentelemetry-exporters-jaeger</artifactId>
+	<version>0.5.0</version>
+</dependency>
+<dependency>
+	<groupId>io.grpc</groupId>
+	<artifactId>grpc-protobuf</artifactId>
+	<version>1.27.2</version>
+</dependency>
+<dependency>
+	<groupId>io.grpc</groupId>
+	<artifactId>grpc-netty</artifactId>
+	<version>1.27.2</version>
+</dependency>
+```
+
+### Gradle
+ 
+#### OpenTelemetry
+```gradle
+compile "io.opentelemetry:opentelemetry-api:0.5.0"
+compile "io.opentelemetry:opentelemetry-sdk:0.5.0"
+compile "io.grpc:grpc-context:1.24.0"
+```
+
+#### LoggerExporter
+```gradle
+compile "io.opentelemetry:opentelemetry-exporters-logging:0.5.0"
+```
+
+#### JaegerExporter
+```gradle
+compile "io.opentelemetry:opentelemetry-exporters-jaeger:0.5.0"
+compile "io.grpc:grpc-protobuf:1.27.2"
+compile "io.grpc:grpc-netty:1.27.2"
+```
+
+### Tracer Configuration
+
+To enable tracing in your OpenTelemetry project configure a Tracer Bean. This bean will be auto wired to controllers to create and propagate spans. This can be seen in the `Tracer otelTracer()` method below. If you plan to use a trace exporter remember to also include it in this configuration file. In [section 3](#instrumentation-using-opentelemetry-contrib-spring-in-progress) we will use an [annotation](#configtracer) to set up this configuration.
```

Comment:
[Question] Are tracer beans a thing from OT? OR from Spring?

Reply:
It's a thing from spring. I used the @Bean annotation to create a OT Tracer object that's managed by Spring. This object is then injected to other classes as a dependency.This seems to be the cleanest way to create and manage spans.

Reply:
Ok, just make it clear since some users may not be familiar with the term form Spring and may incorrectly associated it with OT.